package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "enrise-logger",
   "description": "Logger used within Enrise projects and module's",
-  "version": "0.2.1",
+  "version": "1.0.0",
```

Comment:
We have a breaking change. So we need to up the major-part, hence 1.0.0

From `semver.org`:
"Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,"